src/main/groovy/com/ofg/twitter/place/extractor/PlacesExtractor.groovy

```diff
@@ -21,8 +22,12 @@ class PlacesExtractor {
 
     Map<String, Optional<Place>> extractPlacesFrom(List<Tweet> tweets) {
         Map<String, Optional<Place>> foundPlaces = new ConcurrentHashMap<>()
-        GParsPool.withPool {
-            tweets.eachParallel { foundPlaces << appendExtractedTweet(it) }
+        String savedCorrelationId = CorrelationIdHolder.get()
+        GParsPool.withPool(1) {
+            tweets.eachParallel {
```

Comment:
Not sure how relevant GPars is, but in the future we should think about better, more transparent support, rather than updating correlation ID manually everywhere.